src/org/abymaly/code/calculator/Calculator.java

```diff
@@ -0,0 +1,35 @@
+package org.abymaly.code.calculator;
+
+import java.util.*;
+
+public class Calculator {
+
+    public Operation checkOperator(String strOperator) {
+        Operation currentOperator = null;
+        for (Operation op : Operation.values()) {
+            if (strOperator.equals(op.mark)) {
+                currentOperator = op;
+            }
+        }
+        return currentOperator;
+
+    }
+
+    public double calculator(String input) {
+
+        String[] inputValue = input.split(" ");
+
+        Stack<Double> stack = new Stack<>();
```

Comment:
Nazwa `stack` jest zła. Nawet `stos_liczb_do_przetwarzania_przez_kalkulator` jest dużo lepsza. Jak doszedłem do instrukcji `stack.push` to musiałem wrócić do tego "co to jest stack???", to się samo nie czyta.